server.js

```diff
@@ -1,39 +1,126 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
-// 
+//
 // import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
+// import avocadoSalesData from "./data/avocado-sales.json";
+import booksData from "./data/books.json";
 // import netflixData from './data/netflix-titles.json'
 // import topMusicData from './data/top-music.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+// mongodb://localhost/project-mongo
+
+const dbpass = process.env.DB_PASS;
+const dbuser = process.env.DB_USER;
+
+const mongoUrl =
+  process.env.MONGO_URL ||
+  `mongodb+srv://${dbuser}:${dbpass}@cluster0.1imwj.mongodb.net/Database?retryWrites=true&w=majority`;
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const Book = mongoose.model("Book", {
+  bookID: Number,
```

Comment:
You will be provided with _id so it is no need to pass this one to the database 🤓